frontend/src/components/@common/Map/OverlayMarker.tsx

```diff
@@ -28,10 +27,12 @@ function OverlayMarker({ celeb, restaurant, map, quadrant }: OverlayMarkerProps)
   const ref = useRef();
   const [hoveredId] = useHoveredRestaurantState(state => [state.id]);
   const { isMobile } = useMediaQuery();
+  const [setPreview] = useMapState(state => [state.setPreview]);
```

Comment:
이건 그냥 요렇게 써주어도 될 것 같아요~!
```suggestion
  const setPreview = useMapState(state => state.setPreview);
```